contracts/modules/credit/LineOfCredit.sol

```diff
@@ -99,6 +96,13 @@ contract LineOfCredit is ILineOfCredit, MutualConsent, ReentrancyGuard {
         _;
     }
 
+    modifier excludeEth(address token_) {
+        if (token_ == Denominations.ETH) {
+            revert EthSupportDisabled();
+        }
+        _;
+    }
```

Comment:
We need to include the check in a modifier to prevent the mutual consent from being created.